content/backstage/plugins/jenkins.md

```diff
@@ -12,54 +12,85 @@ seo:
     Build, test, and deploy on Jenkins CI/CD platform.
 
 logoImage: '../../assets/logos/jenkins/logo-jenkins.png'
-coverImage: '../../assets/jenkins-plugin.png'
+coverImage: '../../assets/backstage/plugins/jenkins/jenkins-plugin.png'
```

Comment:
Would you mind switching out the screenshots for light mode ones please? roadie.io only supports light mode and dark mode screenshots look out of place on the white background. 

You can switch your Backstage instance to light mode in the settings.

Reply:
Ah good point. Will keep this in mind for the mind for future work.